src/pages/en/guides/server-side-rendering.md

```diff
@@ -12,9 +12,18 @@ i18nReady: true
 
 ## Enabling SSR in Your Project
 
-To enable SSR you need to use an adapter. This is because SSR requires a server _runtime_: the environment that runs your server-side code. This runtime provides an API that your server-side code can use.
+During development, you can enable SSR by adding the `output: server` configuration option:
 
-Installing an adapter gives Astro access to the corresponding API, and allows Astro to output a script that runs your project on that kind of server.
+    ```js ins={5}
+    // astro.config.mjs
+    import { defineConfig } from 'astro/config';
+
+    export default defineConfig({
+      output: 'server'
+    });
+    ```
+
+To deploy an SSR project, you also need to add an adapter. This is because SSR requires a server _runtime_: the environment that runs your server-side code. Each adapter allows Astro to output a script that runs your project on a specific runtime.
```

Comment:
I'm not entirely sure about moving this comment to the bottom of the example, and having the example only show the `output: 'server'` line. Do you think it might pre-empt some problems to mention needing an adapter above the example, then *also* showing the adapter installed in the config file?

It's not terrible, and I get that the server part is *really* the generic "this is what makes it SSR" thing. But if no one is ever going to have that without *also* having an adapter, then I think I lean towards having the example show both together, even if we use a placeholder or example adapter in there. (vs this, which is to not show an adapter at all so that we don't have to make the call which one to show).

Reply:
What I wanted was to show that you don't need an adapter during development, which wasn't clear before.

The idea is that there _is_ a use case of enabling `output: server` without an adapter: when you know you want SSR, but you don't yet know where you'll deploy.

I think I can find a cleaner way to say this, though